<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SideBar</class>
 <widget class="QWidget" name="SideBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="actionBox">
     <property name="title">
      <string>Actions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="restartButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Restart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shutdownButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Shutdown</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reseedButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Reseed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sliderLabel">
        <property name="text">
         <string>Refresh Speed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="refreshSpeedSlider">
        <property name="minimum">
         <number>500</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="sliderPosition">
         <number>5000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="generalInfoBox">
     <property name="title">
      <string>General Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="statusText">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="uptimeLabel">
          <property name="text">
           <string>Uptime:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="uptimeText">
          <property name="text">
           <string>?</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="versionLabel">
          <property name="text">
           <string>Version: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="versionText">
          <property name="text">
           <string>?</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="netStatusLabel">
          <property name="text">
           <string>Network:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="netStatusText">
          <property name="text">
           <string>?</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="bwInText">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="bwInLabel">
          <property name="text">
           <string>Inbound:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="bwOutText">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="bwOutLabel">
          <property name="text">
           <string>Outbound:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="participatingText">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="participatingLabel">
          <property name="text">
           <string>Part. tunnels:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="netdbBox">
     <property name="title">
      <string>Network Database</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="activePeersLabel">
          <property name="text">
           <string>Active:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="activePeersText">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="fastPeersLabel">
          <property name="text">
           <string>Fast:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="fastPeersText">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="highCapPeersLabel">
          <property name="text">
           <string>High capacity:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="highCapPeersText">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="reseedingLabel">
          <property name="text">
           <string>Reseeding:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="reseedingText">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="knownPeersLabel">
          <property name="text">
           <string>Known:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="knownPeersText">
          <property name="text">
           <string>?</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
